modules/Schedulers/_AddJobsHere.php

```diff
@@ -777,6 +778,11 @@ function aorRunScheduledReports()
             if (empty($scheduledReport->aor_report_id)) {
                 continue;
             }
+            $queued = $db->fetchOne("SELECT count(*) cnt FROM job_queue WHERE data=".$db->quoted($scheduledReport->id)." and deleted=0 and status = 'running' and execute_time >= DATE_ADD(now(), INTERVAL -6 hour)");
```

Comment:
This won't work in SQL server, you would need to use the conversions at least with DATE_ADD here
plus 6 hours is kind of random